code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -13,10 +13,10 @@
 	if(isturf(loc))
 		var/turf/T = loc
 		if(T.get_lumcount() < 0.6)
-			if(!stats.getPerk(PERK_NIGHTCRAWLER))
-				tally += 0.5
-			else
+			if(stats.getPerk(PERK_NIGHTCRAWLER))
 				tally -= 0.5
+			else if(see_invisible != SEE_INVISIBLE_NOLIGHTING)
+				tally += 0.5
```

Comment:
What is the purpose of this?

Reply:
if you don't have night vision you become slow in the dark.